src/controller/tests/TestReadChunking.cpp

```diff
@@ -78,6 +78,7 @@ DECLARE_DYNAMIC_CLUSTER_LIST_BEGIN(testEndpointClusters)
 DECLARE_DYNAMIC_CLUSTER(TestCluster::Id, testClusterAttrs), DECLARE_DYNAMIC_CLUSTER_LIST_END;
 
 DECLARE_DYNAMIC_ENDPOINT(testEndpoint, testEndpointClusters);
+chip::DataVersion gtestEndpointDataVersions[ArraySize(testEndpointClusters)];
```

Comment:
This part can go away with the most recent update to #14471.

Reply:
would rebase after 14471 is merged